addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightV3RGBW.java

```diff
@@ -0,0 +1,267 @@
+                break;
+            case 3:
+                // message whiteMode RGBW bulb channel 3
+                messageBytes = new byte[] { (byte) 0xC9, 0x00, 0x55 };
+                break;
+            case 4:
+                // message whiteMode RGBW bulb channel 4
+                messageBytes = new byte[] { (byte) 0xCB, 0x00, 0x55 };
+                break;
+        }
+        setPower(true, state);
+
+        sendQueue.queue(messageBytes, uidc(type_offset, CAT_WHITEMODE), true);
+    }
+
+    @Override
+    public void nightMode(MilightThingState state) {
+        logger.debug("milight: sendNightMode");
+        byte messageBytes[] = null;
+        byte messageBytes2[] = null;
+        switch (zone) {
+            case 0:
+                // message nightMode all RGBW bulbs
+                messageBytes = new byte[] { 0x41, 0x00, 0x55 };
+                messageBytes2 = new byte[] { (byte) 0xC1, 0x00, 0x55 };
+                break;
+            case 1:
+                // message nightMode RGBW bulb channel 1
+                messageBytes = new byte[] { 0x46, 0x00, 0x55 };
+                messageBytes2 = new byte[] { (byte) 0xC6, 0x00, 0x55 };
+                break;
+            case 2:
+                // message nightMode RGBW bulb channel 2
+                messageBytes = new byte[] { 0x48, 0x00, 0x55 };
+                messageBytes2 = new byte[] { (byte) 0xC8, 0x00, 0x55 };
+                break;
+            case 3:
+                // message nightMode RGBW bulb channel 3
+                messageBytes = new byte[] { 0x4A, 0x00, 0x55 };
+                messageBytes2 = new byte[] { (byte) 0xCA, 0x00, 0x55 };
+                break;
+            case 4:
+                // message nightMode RGBW bulb channel 4
+                messageBytes = new byte[] { 0x4C, 0x00, 0x55 };
+                messageBytes2 = new byte[] { (byte) 0xCC, 0x00, 0x55 };
+                break;
+        }
+        // nightMode for RGBW bulbs requires second message 100ms later.
+        sendQueue.queue(messageBytes, uidc(type_offset, CAT_NIGHTMODE1), false);
+        sendQueue.queue(messageBytes2, uidc(type_offset, CAT_NIGHTMODE2), false);
+    }
+
+    @Override
+    public void setColorTemperature(int color_temp, MilightThingState state) {
+
+    }
+
+    @Override
+    public void setBrightness(int value, MilightThingState state) {
+        if (value <= 0) {
+            setPower(false, state);
+            return;
+        }
+
+        int newCommand = (int) Math.ceil((value * rgbwLevels) / 100.0) + 1;
+        setPower(true, state);
+
+        byte messageBytes[] = new byte[] { 0x4E, (byte) newCommand, 0x55 };
+        logger.debug("milight: send dimming packet '{}' to RGBW bulb channel '{}'", messageBytes, zone);
+        sendQueue.queue(messageBytes, uidc(type_offset, CAT_BRIGHTNESS_SET), true);
+
+        state.brightness = value;
+    }
+
+    @Override
+    public void changeBrightness(int relative_brightness, MilightThingState state) {
```

Comment:
changeBrightness isn't used for these bulbs since the brightness setting is absolute.